lib/xdrgen/generators/go.rb

```diff
@@ -389,7 +389,16 @@ def render_typedef_encode_to_interface(out, typedef)
         name = name(typedef)
         type = typedef.declaration.type
         out.puts "// EncodeTo encodes this value using the Encoder."
-        out.puts "func (s #{name}) EncodeTo(e *xdr.Encoder) error {"
+
+        if type.is_a?(AST::Typespecs::Opaque) and type.fixed?
```

Comment:
❗ Avoid using `and`. It's generally avoided in ruby applications because `&&` and `and` have different order of precedence which catches people up. In this specific case it makes no difference, but `and`s have a way of multiplying. See https://github.com/rubocop/ruby-style-guide#andor.
```suggestion
        if type.is_a?(AST::Typespecs::Opaque) && type.fixed?
```

Reply:
Thanks. This is the first time I code in Ruby. Will fix.

Reply:
Done